pyccel/parser/semantic.py

```diff
@@ -649,6 +412,8 @@ def create_new_function_scope(self, name, decorators):
             name = DottedName(self._current_function, name)
         self._current_function = name
 
+        return child
```

Comment:
Please update the docstring with a `Returns` section

Reply:
Fixed [ec774d4](https://github.com/pyccel/pyccel/pull/1068/commits/ec774d4b4b774aef629d12d90ac4d0b3e8dfa097)